src/com/synopsys/integration/pipeline/scm/GitStage.groovy

```diff
@@ -39,6 +48,56 @@ class GitStage extends Stage {
         getPipelineConfiguration().getScriptWrapper().executeCommandWithException("${gitPath} reset --hard ${githubBranchModel.getBranchName()}")
     }
 
+    void determineAndSetBranch() {
+        getPipelineConfiguration().getLogger().info("In determineAndSetBranch")
```

Comment:
Do you still want this log message? Or do you want to update the message at least?

Reply:
Thought I got all my debugging messages. Thank you for catching.